### PR TITLE
Moving console to lua layer

### PR DESCRIPTION
**Pull request by andrsd**
Refs #9

### CODE REVIEW

**Review by ragusa — APPROVED** [review]

**Review by wdhawkins — APPROVED** [review]

File: lua/framework/lua_app.cc [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+int
+LuaApp::Run(int argc, char** argv)
+{
+  InitPetSc(argc, argv);
+  opensn::Initialize();
+  console.PostMPIInfo(opensn::mpi.location_id, opensn::mpi.process_count);
+
+  int error_code = 0;
+  ParseArguments(argc, argv);
+  if (!termination_posted_)
```
Comment (zhardy-lanl) [inline lua/framework/lua_app.cc:70]:
```suggestion
  if (not termination_posted_)
```

File: lua/framework/lua_app.cc [hunk 2]
```diff
@@ -0,0 +1,262 @@
...
+        {
+          std::cerr << "Invalid option used with command line argument "
+                       "-v. Options are 0,1 or 2."
+                    << std::endl;
+          Exit(EXIT_FAILURE);
+        }
+      }
+
+    } //-v
+    else if ((argument.find('=') == std::string::npos) and (!input_file_found))
```
Comment (zhardy-lanl) [inline lua/framework/lua_app.cc:145]:
```suggestion
    else if ((argument.find('=') == std::string::npos) and (not input_file_found))
```

**Review by zhardy-lanl — APPROVED** [review]
Please make the requested syntax changes. Other than that, good to go.

File: lua/framework/lua_app.cc (hunk 1 above)
Comment (andrsd) [reply lua/framework/lua_app.cc:70]: I need to put this on a postit note and glue that on my monitor. I forgot about this again 🤦‍♂️

**Pull request merged**